flutter-idea/src/io/flutter/analytics/FlutterAnalysisServerListener.java

```diff
@@ -342,51 +367,85 @@ public void serverStatus(AnalysisStatus analysisStatus, PubStatus pubStatus) {
     assert analysisStatus != null;
     if (!analysisStatus.isAnalyzing()) {
       @NotNull HashMap<String, Integer> errorCounts = getTotalAnalysisErrorCounts();
-      int errorCount = 0;
+      errorCount = 0;
       errorCount += extractCount(errorCounts, AnalysisErrorType.CHECKED_MODE_COMPILE_TIME_ERROR);
       errorCount += extractCount(errorCounts, AnalysisErrorType.COMPILE_TIME_ERROR);
       errorCount += extractCount(errorCounts, AnalysisErrorType.SYNTACTIC_ERROR);
-      FlutterInitializer.getAnalytics().sendEventMetric(DAS_STATUS_EVENT_TYPE, ERRORS, errorCount); // test: serverStatus()
-      int warningCount = 0;
+      warningCount = 0;
       warningCount += extractCount(errorCounts, AnalysisErrorType.STATIC_TYPE_WARNING);
       warningCount += extractCount(errorCounts, AnalysisErrorType.STATIC_WARNING);
-      FlutterInitializer.getAnalytics().sendEventMetric(DAS_STATUS_EVENT_TYPE, WARNINGS, warningCount); // test: serverStatus()
-      int hintCount = extractCount(errorCounts, AnalysisErrorType.HINT);
-      FlutterInitializer.getAnalytics().sendEventMetric(DAS_STATUS_EVENT_TYPE, HINTS, hintCount); // test: serverStatus()
-      int lintCount = extractCount(errorCounts, AnalysisErrorType.LINT);
-      FlutterInitializer.getAnalytics().sendEventMetric(DAS_STATUS_EVENT_TYPE, LINTS, lintCount); // test: serverStatus()
+      hintCount = extractCount(errorCounts, AnalysisErrorType.HINT);
+      lintCount = extractCount(errorCounts, AnalysisErrorType.LINT);
+      if (IS_TESTING) {
+        errorCount = warningCount = hintCount = lintCount = 1;
+      }
+      maybeReportErrorCounts();
```

Comment:
have you manually verified that these #s match what the analyzer window shows?

Reply:
It's been a while, but yes.